### PR TITLE
Update Readme to French language

### PR DESCRIPTION
**Pull request by akria18**
**What this PR does / why we need it**:
this is a PR for updating the README.md to french language
**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Checklist:**
-   [ ] Fixes #<issue number>
-   [ ] Signed the commit for DCO to be passed
-   [ ] Labelled this PR & related issue with `documentation` tag

### CODE REVIEW

**Review by ksatchit — DISMISSED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -12,92 +13,95 @@
 [![Open Source Love](https://badges.frapsoft.com/os/v2/open-source.svg?v=103)](https://github.com/litmuschaos/litmus-docs)
 ![GitHub top language](https://img.shields.io/github/languages/top/litmuschaos/litmus-docs)
 
-Additional details on the Docusaurus project can be found [here](https://docusaurus.io/docs/en/installation.html).
 
-## For Developers
+Une description détaillée de porjet Docusaurus est disponible
```
Comment (divya-mohan0209) [inline README.md]: Shouldn't project be projet in French?

**Review by divya-mohan0209 — COMMENTED** [review]
LGTM except the minor spelling mistake. Please make the change & resubmit it for review, if possible.

**Review by akria18 — COMMENTED** [review]
fixed error. Thank you

File: README.md (hunk 1 above)
Comment (akria18) [reply README.md]: Yes,  done. Thank you

**Pull request merged**